versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 2.7.0"
+      region  = var.region
+    }
+  }
+}
```

Comment:
The code in this `required_providers` block is not supported in terraform v0.12.x, and throws a warning which might throw some people off.  What I believe I would suggest is to basically do what you did here and add your own `versions.tf` file and then do a `terraform init` with terraform v0.13.x or greater and it should work fine.